Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/DocumentProvider.kt

```diff
@@ -111,67 +112,54 @@ class DocumentProvider : DocumentsProvider() {
         parentDocumentId: String,
         mimeType: String,
         displayName: String
-    ): String {
+    ): String? {
+        rootDirectory ?: return null
+
         val folder = documentIdToPath(parentDocumentId)
         val file = findFileNameForNewFile(File(folder, displayName))
         if (mimeType == DocumentsContract.Document.MIME_TYPE_DIR) {
             file.mkdirs()
         } else {
             file.createNewFile()
         }
+        Log.w("DOCUMENTPROVIDER", file.absolutePath)
```

Comment:
This isn't particularly useful currently. Would you want to prefix it with "Created file <file>" or just get rid of it?

Reply:
Oops, that's a debugging leftover.

Reply:
Removed.